paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/PlaygroundSettings.kt

```diff
@@ -141,14 +141,9 @@ internal class PlaygroundSettings private constructor(
         private const val sharedPreferencesKey = "json"
 
         fun createFromDefaults(): PlaygroundSettings {
-            val settings = allSettingDefinitions.mapNotNull {
-                val saveable = it.saveable()
-                if (saveable != null) {
-                    it to MutableStateFlow(saveable.defaultValue)
-                } else {
-                    null
```

Comment:
This was the bug. We need to have a value for all settings, including things that aren't `saveable`.

Reply:
Otherwise we don't loop over them and do their side effects for things like `configure`.